pkg/context/helpers.go

```diff
@@ -135,6 +135,41 @@ func (c *TestContext) AddTestDirectory(srcDir, destDir string) {
 	}
 }
 
+func (c *TestContext) AddTestDriver(src, name string) string {
+	c.T.Helper()
+
+	data, err := ioutil.ReadFile(src)
+	if err != nil {
+		c.T.Fatal(err)
+	}
+
+	dirname, err := c.FileSystem.TempDir("", "porter")
+	if err != nil {
+		c.T.Fatal(err)
+	}
+
+	// filename in accordance with cnab-go's command driver expectations
+	filename := fmt.Sprintf("%s/cnab-%s", dirname, name)
+
+	newfile, err := c.FileSystem.Create(filename)
+	if err != nil {
+		c.T.Fatal(err)
+	}
+
+	if len(data) > 0 {
+		newfile.Write(data)
```

Comment:
Write can return an error and so can Chmod and Close. Let's use `require.NoError(c.T, err)` to check that they didn't return an error. 

The other helpers in porter use testify so it's okay to make use of that here as well.